db/Dockerfile

```diff
@@ -9,11 +9,12 @@ RUN apt-get update \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-ADD make_db.sh /docker-entrypoint-initdb.d/
-ADD setup-wale.sh /docker-entrypoint-initdb.d/
-COPY docker-entrypoint1.sh /
-RUN chmod +x /docker-entrypoint1.sh
```

Comment:
I think we should keep this line: https://github.com/mattermost/mattermost-docker/commit/808f5af6698fbbb39a73229a4445a19da3507527